src/library/styles/createStyledComponent.js

```diff
@@ -1,31 +1,76 @@
 /* @flow */
-import glamorous from 'glamorous';
+
+import styled from 'react-emotion';
+import withPropsFn from 'recompose/withProps';
 import componentStyleReset from './componentStyleReset';
+import isValidProp from '../utils/isValidProp';
+import { isProduction } from '../utils/nodeEnv';
 
 export default function createStyledComponent(
   element:
     | React$StatelessFunctionalComponent<*>
     | React$ComponentType<*>
     | string,
-  styles: Object | ((props: Object, context?: Object) => Object),
-  options?: Object = {}
+  styles:
+    | Object
+    | Array<Object>
+    | ((props: Object, context?: Object) => Object | Array<Object>),
+  options?: {
+    displayName?: string,
+    filterProps?: Array<string>,
+    forwardProps?: Array<string>,
+    includeStyleReset?: boolean,
+    rootEl?: string,
+    withProps?: Object
+  } = {}
 ) {
-  const { includeStyleReset, ...restOptions } = options;
-  let outStyles;
+  const {
+    displayName,
+    filterProps = [],
+    forwardProps = [],
+    rootEl,
+    withProps
+  } = options;
+  const includeStyleReset = options.includeStyleReset || false;
```

Comment:
I assume flow required this? Curious as to why...

Reply:
Nope...left from initial debuggerating... Fixed, thanks!